boards/posix/nrf52_bsim/find_babblesim.cmake

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
```

Comment:
Good direction, but I would like this to become a CMake module.

That means relocate this file to https://github.com/zephyrproject-rtos/zephyr/tree/main/cmake/modules and name it:
`Findbabblesim.cmake`.

That way users can lookup babblesim using:
```
find_package(babblesim)
```

let's take other details offline.

Reply:
The cmake changes will be taken in another PR.